.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Could you add this file to the gitignore? I don't like editor-specific settings in my repositories 😀

Reply:
Sure. I need this file because my prettier rules are different and they're applied on each save. I'll keep it only in my fork.